deps/v8/src/base/cpu.cc

```diff
@@ -676,8 +676,14 @@ CPU::CPU()
 
 #ifndef USE_SIMULATOR
 #if V8_OS_LINUX
+#if V8_GLIBC_PREREQ(2, 16)
```

Comment:
getauxval is available on musl and other libc's, so this check is wrong, and there is no reasonable way to check it - i just wouldn't bother, the other path is directly equivalent already